src/util/sss_ldap.c

```diff
@@ -279,10 +281,21 @@ static void sss_ldap_init_sys_connect_done(struct tevent_req *subreq)
     }
 
     if (ldap_is_ldaps_url(state->uri)) {
+        ticks_before_install = get_watchdog_ticks();
         lret = ldap_install_tls(state->ldap);
+        ticks_after_install = get_watchdog_ticks();
         if (lret != LDAP_SUCCESS) {
             if (lret == LDAP_LOCAL_ERROR) {
                 DEBUG(SSSDBG_FUNC_DATA, "TLS/SSL already in place.\n");
+            } else if (ticks_after_install > ticks_before_install) {
```

Comment:
I would propose to move this check below `sss_ldap_get_diagnostic_msg()` and messages already printed there. Just for the sake of a chance there will be something useful one day.

Then you can have additional message like "assuming signal interruption".